keyboards/handwired/dtisaac01/readme.md

```diff
@@ -0,0 +1,19 @@
+# dtisaac01
+
+![dtisaac01](imgur.com image replace me!)
+
+*A short description of the keyboard/dtisaac01
```

Comment:
Please fill these out.

Reply:
I filled these in the new patch set.
Please help me check again and merge it.
Thanks.